### DISCUSSION TITLE
有没有方式能显示一个工单已经执行完sql的状态

### DISCUSSION THREAD

**Discussion by dbdartisan** [hhyo/Archery#2540] [Q&A]
比如一个工单有100条sql，比如前面10条执行完了能不能先显示已完成的状态。目前是要全部执行完才能更新。这个好像是需要goinception是这样最后返回的。

**Comment by dbdartisan** [comment]
这是不是只能改成每一条sql调用一次goInception来实现，者楼主觉得合适吗

**Comment by Grain-Yu** [comment]
实现上应该没问题，不过要改掉inception的代码，配合前端页面做刷新。现在的做法是直接调用inception的模块。

**Reply by dbdartisan** [reply]
嗯，好的，不然确实看不到执行进度。

**Comment by hhyo** [comment]
inception可以获取执行进度，如果不拆sql可以考虑加一个整体进度条
